docs/platform_developers/developer_getting_started.rst

```diff
@@ -0,0 +1,151 @@
+********************************
+Developer Getting Started Guide
+********************************
+
+This page describes how to setup your development environment to:
+
+* Setup Inmanta pip artifacts
+* Verify that you already have GitLab SSH keys.
+* Install VS Code and Inmanta extension.
+* Setting up Python virtual environments.
+* Benefit from linting and code navigation by setting up a project.
+* Set project sources
+* Module developers guide
+
+
+Setup Inmanta pip artifacts
+#############################
+
+In order to use Inmanta's internal pip artifacts:
+
+.. code-block:: bash
+
+    mkdir   ~/.config/pip
+
+    nano ~/.config/pip/pip.conf
+
+And below text into your ``pip.conf`` file:
+
+::
+    [global]
+    index_url=https://artifacts.internal.inmanta.com/inmanta/dev
+
+
+Verify that you already setup GitLab SSH keys
+###############################################
+
+The steps to add SSH keys have already been mentioned in ``Inmanta Getting Started Guide`` which was emailed to you.
+You can click `here <https://internal.inmanta.com/topics/starting.html>`_ to read it again.
+
+In order to verify that it is working correctly, run below command:
+
+.. code-block:: bash
+    
+    ssh git@git.inmanta.com
+
+
+You should see your name in the response.
+
+
+Install VS Code and Inmanta extension
+#######################################
+
+This step does not need any additional information but if you need some help, you can always ask a colleague.
```

Comment:
This goes into the public documentation,...

Reply:
I would link here to the vscode documentation and the documentation in the vs code extension

Reply:
Could you please provide the links to those docs?

Reply:
https://github.com/inmanta/vscode-inmanta

Reply:
and for the docs of vscode itself, google is your friend ;-)

Reply:
yes, I meant our own doc  👍🏻 
Thanks